docs/install/upgrade.md

```diff
@@ -60,4 +60,4 @@ So, copying `notebook` and `conf` directory should be enough.
 ### Upgrading from Zeppelin 0.7 to 0.8
 
  - From 0.8, we recommend to use `PYSPARK_PYTHON` and `PYSPARK_DRIVER_PYTHON` instead of `zeppelin.pyspark.python` as `zeppelin.pyspark.python` only effects driver. You can use `PYSPARK_PYTHON` and `PYSPARK_DRIVER_PYTHON` as using them in spark.
- 
\ No newline at end of file
+ - From 0.8, depending on your device, the keyboard shortcut `Ctrl-L` or `Command-L` is not supported. 
```

Comment:
Thanks for suggestions. I just agonized the showing form about what kind of keyboard shortcut is more clear for us, because in my case, `command-l` is good as well. :smile: 
Anyway, I'll rewrite this!